src/google/protobuf/compiler/code_generator.h

```diff
@@ -193,6 +193,9 @@ typedef GeneratorContext OutputDirectory;
 PROTOC_EXPORT void ParseGeneratorParameter(
     const std::string&, std::vector<std::pair<std::string, std::string> >*);
 
+// Strips ".proto" or ".protodevel" from the end of a filename.
```

Comment:
Method placement might be somewhat strange, as compiler part does not have generic utility file (e. g. to common.h or util.h).

Suggestions are welcome.